ezyrb/ann.py

```diff
@@ -122,11 +122,10 @@ def fit(self, points, values):
         """
 
         self._build_model(points, values)
-        self.optimizer = torch.optim.Adam(self.model.parameters())
+        self.optimizer = torch.optim.Adam(self.model.parameters(), lr = 0.01)
```

Comment:
Can we add the possibility to pass the optimizer (Adam by default) and the learning_rate (0.001 by default)?